src/descriptors/power_system_inputs.json

```diff
@@ -259,6 +259,18 @@
         }
     ],
     "generator": [
+        {
+            "unit": "%",
+            "name": "fotr",
+            "description": "Forced Outage Rate",
+            "default_value": 0.0
+        },
+        {
+            "unit": "hours",
+            "name": "mttr",
+            "description": "Mean Time to Recover (in Hours)",
+            "default_value": 100
```

Comment:
I think this should be null

Reply:
Yeah, I made this change in the new commit.